consensus/ibft/ibft.go

```diff
@@ -644,6 +645,15 @@ func (i *Ibft) insertBlock(block *types.Block) error {
 		return err
 	}
 
+	i.logger.Info(
+		"final committed",
```

Comment:
Maybe specify that block is commited?
```suggestion
		"block committed",
```